app/javascript/react/components/Greetings.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreetings } from '../redux/greetings/greetingSlice';
+
+const Greetings = () => {
+  const dispatch = useDispatch();
+  const { loading, greeting, error } = useSelector((state) => state.greetings);
+
+  useEffect(() => {
+    dispatch(fetchGreetings());
+  }, [dispatch]);
+
+  if (loading) {
+    return <p>Loading...</p>;
+  }
+
+  if (error) {
+    return <p>{ error }</p>;
+  }
+
+  return (
+    <p>{ greeting }</p>
+  );
+};
```

Comment:
- [ ] As required for the project, kindly make sure your the two apps work together. Right now, I can have `Greetings or Messages` in database but at your UI is blank. Make sure you fix that. I suggest you first test your app at your side and make sure it works correctly before you submit another review.

| Database messages | Your UI |
| ---------------------- | ---------------- |
|  ![wl](https://user-images.githubusercontent.com/83042742/230249276-29d59668-56c3-44ad-aadb-2693cf9cdf72.gif) | ![Screenshot from 2023-04-05 21-20-39](https://user-images.githubusercontent.com/83042742/230249369-bc758660-17fe-4489-9581-eb7e1a955020.png) |
| ![Screenshot from 2023-04-05 21-05-59](https://user-images.githubusercontent.com/83042742/230249409-7854900b-d39c-450f-8dee-80723261dccc.png) | ![Screenshot from 2023-04-05 21-20-39](https://user-images.githubusercontent.com/83042742/230249469-c77ca908-17fc-4a1b-877a-4f9810e83caa.png) |